docker-compose.yml

```diff
@@ -85,6 +85,21 @@ services:
     depends_on:
       - base
 
+  crashpad-symbolize:
+    build:
+      args:
+        - GITHUB_TAG=${GITHUB_TAG:-24.lts.30}
```

Comment:
These being build args makes it using it a bit awkward, as you have to rebuild the container every time when switching a binary. Probably not an issue for partners, but more for Cobalt dev team.

Lets get this in as-is, but it would be great if someone had time to follow up and turn them into container run-time args rather - let's add a follow-up bug ?

Reply:
I think this is something that pretty much only partners would use, as we would use go/crash which already is symbolized, so I think not having to supply these arguments each time you want to run the symbolization is nice